examples/js/loaders/AMFLoader.js

```diff
@@ -48,18 +48,18 @@ THREE.AMFLoader.prototype = {
 
 	parse: function ( data ) {
 
-		function loadDocument( data ) {
+		function _loadDocument( data ) {
```

Comment:
Right.  AMFLoader returns a Group of Object3Ds with children, though.  You're suggesting not to have a user-configurable default material at all?

Additionally, there is a "special material" id 0 (null material) in AMF.  This material is supposed to represent volumes that are printed from material that, for example, should be removed (supports).  I haven't implemented this yet.  I could implement it as a wire, line or flat shaded material with an alpha but I thought it might be useful to let the user specify.  I could always just create a default "null material" and assume people will figure it out. :P

What do you suggest?